mediawiki/LocalSettings.php

```diff
@@ -285,6 +284,12 @@
 # Maintainers
 $wgGroupPermissions['maintainers'] = $wgGroupPermissions['sysop'];
 
+# Rm-spam 
+$wgGroupPermissions['rm-spam'] = $wgGroupPermissions['user'];
+$wgGroupPermissions['rm-spam']['delete'] = true ;
+$wgGroupPermissions['rm-spam']['block'] =true ;
```

Comment:
nit: space around `=`. I realise the current code quality may not be stellar because it was all copied over from the server, but we should try to hold new code to a higher standard. Cleanups of surrounding code are also encouraged.